applications/tari_testnet_miner/src/t_blake_pow.rs

```diff
@@ -67,7 +67,7 @@ impl TestBlakePow {
                 nonce += 1;
             }
             if nonce == start_nonce {
-                header.timestamp = header.timestamp.checked_add_signed(Duration::milliseconds(1)).unwrap();
+                header.timestamp = header.timestamp + 1;
```

Comment:
Just reset to the current time here.

Reply:
It might better to reset and redo difficulty. Running through u64 should not be fast.

Reply:
I mean, let's say it takes 5 minutes to run through all the nonces. Wouldn't you want to reset the timestamp to the current time?

Or if it took 2 hours and for some reason another bock hadn't been found. Now the timestamp > 2 hrs away from the real time when the block is propagated and it'll be rejected.

Reply:
Block wont be rejected for being 2hr behind in time. They are only rejected if they are in ahead in time.

Reply:
Still think it's more accurate to reset to current time, but not going to bikeshed further